plugins/mqtt/command/read_write.c

```diff
@@ -20,11 +20,20 @@
 #include <assert.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <sys/time.h>
 
 #include "read_write.h"
 
-int command_read_once_request(neu_plugin_t *plugin, neu_json_mqtt_t *mqtt,
-                              neu_json_read_req_t *req, uint32_t req_id)
+static uint64_t current_time()
+{
+    struct timeval tv;
+    gettimeofday(&tv, NULL);
+    uint64_t ms = tv.tv_sec;
+    return ms * 1000 + tv.tv_usec / 1000;
+}
```

Comment:
when the number of messages is large, calling gettimeofday too many times is inefficient